src/features/sarcophagi/components/Rewrap.tsx

```diff
@@ -38,8 +38,22 @@ export function Rewrap() {
 
   const nowMs = Date.now();
 
-  const rewrapIntervalSeconds = sarcophagus?.maximumRewrapInterval?.toNumber() ?? 0;
-  const rewrapIntervalMs = rewrapIntervalSeconds * 1000;
+  const maxRewrapIntervalFromSarcophagusSec = sarcophagus?.maximumRewrapInterval?.toNumber() ?? 0;
+
+  // The calculated max rewrap interval is
+  // ( new resurrection time - previous resurrection time ) * (200 / cursed bond percentage)
+  // Defaults to max possible number
+  const maxRewrapIntervalCalculatedSec = sarcophagus
+    ? (Number(sarcophagus.resurrectionTime) - Number(sarcophagus.previousRewrapTime)) *
+      (200 / sarcophagus.cursedBondPercentage)
```

Comment:
ahh this is the magic calculation -- will get some contract specs added using this